crates/codegen/parser/generator/src/precedence_parser_definition.rs

```diff
@@ -98,15 +98,35 @@ impl PrecedenceParserDefinitionNodeExtensions for PrecedenceParserDefinitionNode
                     version_tag = version_quality_ranges.disambiguating_name_suffix(),
                     name = operator_definition.name().to_snake_case()
                 );
-                operator_closures.push(quote! {
-                    let #closure_name = |stream: &mut Stream|
-                        PrecedenceHelper::to_precedence_result(
-                            RuleKind::#rule_kind,
-                            #left_binding_power,
-                            #right_binding_power,
-                            #operator_code
-                        );
-                });
+                if left_binding_power == 255 {
```

Comment:
nit: I suggest using `u8::MAX` everywhere instead of `255` for readability, or defining a module-level `const` and using it everywhere?

```rust
const MAX_BINDING_POWER = u8::MAX;
```

Reply:
Went one better.